api/src/main/java/io/strimzi/api/kafka/model/KafkaClusterSpec.java

```diff
@@ -172,7 +174,7 @@ public void setImage(String image) {
     }
 
     @JsonInclude(JsonInclude.Include.NON_NULL)
-    @Description("CPU and memory resources to reserve.")
+    @Description("CPU and memory resources to reserve for the Cruise Control container.")
```

Comment:
we are in the `KafkaClusterSpec`, why this is about Cruise Control container resources?